### DISCUSSION TITLE
Missing results in allure-results

### DISCUSSION THREAD

**Discussion by gagatekns** [mmisty/cypress-allure-adapter#104] [Q&A] [answered]
Hi, I have just set up cypress-allure-adapter, before I was using Shelex plugin. 
On both cypress versions where Shelex plugin works, and on the lastest after test run there is no results in allure-results folder.
But this code
```
      on('before:run', details => {
        reporter?.writeEnvironmentInfo({
          info: {
            os: details.system.osName,
            osVersion: details.system.osVersion,
            browser: details.browser?.displayName + ' ' + details.browser?.version,
            ...config.env
          },
        });
```

generate 'environment.properties' under allure-results.
What did I wrong?

**Comment by mmisty** [comment, marked as answer]
@gagatekns Sorry, I don't understand
to run with @mmisty/cypress-allure-adapter I recommend to disable shelex's plugin

If you still have problems please refer to: 
 - https://github.com/mmisty/cypress-allure-adapter/blob/main/docs/gherkin.md
 - https://github.com/mmisty/cypress-allure-adapter-example/tree/main/examples
 
If you still don't find a solution there please attach: 
 - e2e.ts (setup file)
 - cypress.config.ts file (plugins)
 - package.json config
 - env variables that you use (if you use separate cypress.env file
